src/marketing/buttons/button.scss

```diff
@@ -1,117 +1,106 @@
 .btn-mktg {
+  position: relative;
+  z-index: 1;
   display: inline-block;
-  padding: $spacer-3 $spacer-4;
-  font-size: $h5-size;
-  font-weight: $font-weight-semibold;
-  color: var(--color-mktg-btn-text);
+  appearance: none !important;
+  // stylelint-disable-next-line primer/spacing
+  padding: rem(($spacer-3) * 0.9) rem($spacer-4) rem(($spacer-3) * 1.1);
+  // stylelint-disable-next-line primer/typography
+  font-size: 1rem;
+  font-weight: $font-weight-bold;
+  line-height: 1;
   white-space: nowrap;
   vertical-align: middle;
-  cursor: pointer;
   user-select: none;
-  background-color: var(--color-mktg-btn-bg);
-  border: $border-width $border-style var(--color-mktg-btn-border);
-  border-radius: $border-radius;
-  transition-duration: $transition-time / 2;
-  transition-property: background-color, border-color, box-shadow;
-  appearance: none; // Corrects inability to style clickable `input` types in iOS.
+  border: 0;
+  // stylelint-disable-next-line primer/borders
+  border-radius: rem(6px);
+
+  @include btn-solid-mktg(
+    var(--color-text-white),
+    lighten($primary-blue-mktg, 5%),
+    $primary-blue-mktg,
+    lighten($secondary-blue-mktg, 5%),
+    $secondary-blue-mktg
```

Comment:
I think this could also be replaced with variables from primitives. Then the SASS variables [here](https://github.com/primer/css/pull/1352/files#diff-cb94cf5afa0ea8c52869cc8fcbf1c89736a5c69629624347f62b924da09ac590) shouldn't be needed anymore.

Reply:
Great point! I fixed this instance now ✨